"use strict";

console.log("chapitre 14");

// Fonction asynchrone
// Levée et récupération d'erreur à l'intérieur du fonction asynchrone
// Exercice : reprendre l'exercice du chapitre précédent et utiliser les mot-clés async/await
